docs/developer-guide/rosplane/estimator/estimator-example.md

```diff
@@ -0,0 +1,373 @@
+# Estimator Example
+
+<!-- TODO: rename this to continuous discrete once full state is integrated. -->
+
+## Overview
+
+The `estimator_example` class implements a continuous-discrete Kalman filter as described in section 8.5 of the [UAV book](https://github.com/randybeard/mavsim_public) or 8.6 and 8.7 of volume one of the same book.
+Specifically, this estimator uses algorithm 2 of chapter 8 in Volume 1.
+It utilizes a two stage estimation process along with low pass filtering the inversion of a few sensor models and direct measurements.
+The roll and pitch of the aircraft are estimated first.
+This is called the attitude estimation step though not all of the attitude is estimated here.
+The other states are then estimated as a all at once.
+This is called the position estimation step, though more than just position is estimated during this step.
+The estimator runs on a set timer with a configurable frequency (see Parameters section for details).
+
+The estimator makes large use of something called a pseudo-measurements.
+These measures assume that the side slip angle is zero and the corresponding wind triangle solution.
+This allows us to take these measures and then find the wind in the north and east directions along with the yaw of the aircraft.
+They are more fully explored in section 8.11.4 of the UAV book.
+
+## Nomenclature
+
+| Symbol | Meaning | Range |
+|:------:|:-------:| :---: |
+|$\large{\chi}$| Course | $[-\pi,\pi)$ |
+|$\large{\phi}$| Roll | $[-\pi,\pi)$ |
+|$\large{\theta}$| Theta | $[-\pi,\pi)$ |
+|$\large{\psi}$| Yaw | $[-\pi,\pi)$ |
+|$\large{h}$| Altitude | - |
+|$\large{p}$| Roll Rate | - |
+|$\large{q}$| Pitch Rate | - |
+|$\large{r}$| Yaw Rate | - |
+|$\large{V_a}$| Airspeed | $\geq 0$ |
+|$\large{w_n}$| Wind North | - |
+|$\large{w_e}$| Wind East | - |
+
+## Sensor Model Inversion
+
+The roll, pitch and yaw rates are directly measured by the rate gyro and low pass filtered.
```

Comment:
and -> and then low pass filtered. Otherwise can sound like the low-pass filter and the rate gyro is doing the measurement.